CHANGELOG.md

```diff
@@ -1,7 +1,46 @@
 # Changelog
-
 All notable changes to this project will be documented in this file.
 
+## 3.0.0 - 2024-05-02
+[Full Changelog](https://github.com/nextcloud-libraries/nextcloud-l10n/compare/v2.2.0...v3.0.0)
+
+### Breaking changes
+Instead of also sanitizing the replacing variables, now only the result is sanitized, see [pull request #648](https://github.com/nextcloud-libraries/nextcloud-l10n/pull/648).
```

Comment:
While I see the usecase, it's very problematic when a string contains 2 inputs that are user input 🙈
Maybe a second option would be better? 🤔

Reply:
We can also add e.g. this API:

```js
const userinput = '...'
t(
    'foo',
    '{a}link to {userinput}{end_a}',
    {
        a: {
            value: '<a>',
            escape: false,
        },
        end_a: {
            value: '</a>',
            escape: false,
        },
        userinput,
    },
)
```